my_game.py

```diff
@@ -394,7 +394,7 @@ def __init__(self):
 
         # Variable that will hold a list of shots fired by the player
         self.player_shot_list = None
-        self.player_shot_fire_rate_counter = 0
+        self.player_shot_fire_rate_timer = 0
```

Comment:
You need to change the name everywhere it is used.

Reply:
It is done, I've just pushed it Tobias.